pipeline/notifications/rating_reminder.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from pipeline.notifications.notifier import Notifier
+from pipeline.types import NotificationInfo, NotificationInput, NotificationType
+
+
+class RatingReminderNotifier(Notifier):
+    def __init__(self):
+        pass
+
+    def get_notifications(
+        self, inp: NotificationInput
+    ) -> List[NotificationInfo]:
+        remind_users = []
+        min_chat: int = 3
+        numMessages = 0
+        listUsers = inp.chatdata.participants
+        chatMessages = inp.chatdata.messages
+        dateTime = inp.run_datetime
+        for messagesSent in chatMessages:
+            if messagesSent.from_user in listUsers:
+                numMessages += 1
+        if dateTime.weekday() == 5 and numMessages >= min_chat:
+            for uid, user in listUsers.items():
+                remind_users.append(
+                    NotificationInfo(
+                        recipient=user,
+                        notification_type=NotificationType.RATE_MATCH,
+                    )
+                )
+            return remind_users
+        else:
```

Comment:
You don't need multiple return statements at all. You can just return `remind_users` because if the condition is not met, then it will remain empty.

For future cases, when you have an if block that returns, there is no need for an else block, since if that condition is met, the function will return. If the condition is not met, it will proceed.

Instead you can write:

```python
if condition:
  return value
return value_otherwise
```